src/conduits/java/com/enderio/conduits/common/blockentity/ConduitBlockEntity.java

```diff
@@ -91,7 +94,8 @@ public void updateClient() {
     /**
      * Handle a connection state update from the client.
      */
-    @UseOnly(LogicalSide.SERVER)
+
```

Comment:
I think it adds an extra whitespace?

Reply:
I think it did it everywhere where the annotation is used? Does our style guide not catch that?

Reply:
yeah, happened when I've pasted the new annotations in, I'm surprised the style checker didn't catch that. I've fixed them now